### ISSUE TITLE
Inconsistant limittype specification

### ISSUE THREAD

**Issue by qbojj** [KhronosGroup/Vulkan-Docs#2461]
Let us assume that if the limittype contains min for limit alpha then: `device->alpha >= limit->alpha` (similar holds for max)

As for:
- `minMemoryMapAlignment` is `min,pot` then `device->minMemoryMapAlignment >= limit`
- `minStorageBufferOffsetAlignment` is also `min,pot` so `device->minStorageBufferOffsetAlignment >= limit`

But as `minMemoryMapAlignment` restricts the device (the higher the limit the better), but `minStorageBufferOffsetAlignment` restricts the user (the lower the better) maybe (even if it sounds unintuitive) the limits that are restricting the user should have reversed limittypes.

Then we would have `minMemoryMapAlignment`: `min,pot`, but `minStorageBufferOffsetAlignment` would be `max,pot`, but that would align with the device/user restriction reasoning.

_Originally posted by @qbojj in https://github.com/KhronosGroup/Vulkan-Profiles/issues/703#issuecomment-2456966670_

**Comment by r-potter** [comment]
These do seem inconsistent. The `Required Limits` table in the spec defines `minMemoryMapAlignment` as >= 64, but `minStorageBufferOffsetAlignment` <= 256.

**Comment by qbojj** [comment]
Maybe I should have sent this issue to the XML repo, as `minMemoryMapAlignment` >= 64, but `minStorageBufferOffsetAlignment` <= 256 is what I would expect (as stated in the introduction), but the xml sets the limittype to the same value (`min,pot`)
```xml
            <member limittype="bits"><type>uint32_t</type>              <name>viewportSubPixelBits</name><comment>number bits of subpixel precision for viewport</comment></member>
            <member limittype="min,pot"><type>size_t</type>             <name>minMemoryMapAlignment</name><comment>min required alignment of pointers returned by MapMemory (bytes)</comment></member>
            <member limittype="min,pot"><type>VkDeviceSize</type>       <name>minTexelBufferOffsetAlignment</name><comment>min required alignment for texel buffer offsets (bytes) </comment></member>
            <member limittype="min,pot"><type>VkDeviceSize</type>       <name>minUniformBufferOffsetAlignment</name><comment>min required alignment for uniform buffer sizes and offsets (bytes)</comment></member>
            <member limittype="min,pot"><type>VkDeviceSize</type>       <name>minStorageBufferOffsetAlignment</name><comment>min required alignment for storage buffer offsets (bytes)</comment></member>
            <member limittype="min"><type>int32_t</type>                <name>minTexelOffset</name><comment>min texel offset for OpTextureSampleOffset</comment></member>
            <member limittype="max"><type>uint32_t</type>               <name>maxTexelOffset</name><comment>max texel offset for OpTextureSampleOffset</comment></member>
```

**Comment by oddhack** [comment]
> Maybe I should have sent this issue to the XML repo

Vulkan-Docs holds the canonical version of vk.xml, if that's what you mean by "XML repo".

**Comment by oddhack** [comment]
This should be fixed in the 1.4.304 spec update.
